app/lib/common/widgets/html_editor/html_editor.dart

```diff
@@ -272,10 +280,17 @@ class HtmlEditorState extends State<HtmlEditor> {
           autoFocus: widget.autoFocus,
           header: widget.header,
           // local states
-          editorScrollController: editorScrollController,
+          editorScrollController:
+              widget.scrollController ?? editorScrollController,
           editorState: editorState,
           editorStyle: desktopEditorStyle(),
           footer: generateFooter(),
+          characterShortcutEvents: [
+            ...standardCharacterShortcutEvents,
+            if (widget.roomId != null)
+              ...mentionShortcuts(context, widget.roomId!),
```

Comment:
we are trying to avoid `!` at most cost... so can we maybe have a local variable `roomId = widget.roomId` and then just do:

```suggestion
            if (roomId != null)
              ...mentionShortcuts(context, widget.roomId),
```